packages/expo-error-recovery/src/ExpoErrorRecovery.web.ts

```diff
@@ -3,9 +3,14 @@ const LOCAL_STORAGE_KEY = 'EXPO_ERROR_RECOVERY_STORAGE';
 
 function _consumeRecoveryProps(): string | null {
   if (!canUseDOM) return null;
-  const props = localStorage.getItem(LOCAL_STORAGE_KEY);
-  localStorage.removeItem(LOCAL_STORAGE_KEY);
-  return props;
+  try {
+    const props = localStorage.getItem(LOCAL_STORAGE_KEY);
+    localStorage.removeItem(LOCAL_STORAGE_KEY);
+    return props;
+  } catch (e) {
```

Comment:
I don't know if silencing this error is best. At the very least check the type and throw if it's not `SecurityError`.